### ISSUE TITLE
Bring up to date with 2.12.15?

### ISSUE THREAD

**Issue by mrweiner** [hfischberg/v2Recurly-client#1]
@hfischberg There have been a couple of commits since this repo was created, that bring the lib up to date with API v2.28. Any possibility of getting those changes pulled into here?

**Comment by hfischberg** [comment]
@mrweiner Hi there.

This github repo was created simply as a stop gap intiially some months ago.

We cannot update this and recommend you review/move to either the newest PHP V2 client at the link below, or review the V3 PHP:

V2: https://github.com/recurly/recurly-client-php/tree/v2

V3: https://github.com/recurly/recurly-client-php

Please let me know if you have any questions!

Best,

**Comment by mrweiner** [comment]
@hfischberg right, the reason this was created initially was that I need to run v2 and v3 simultaneously, but both v2 and v3 of the API live on the same packagist package (https://packagist.org/packages/recurly/recurly-client). This means that it's impossible for me to `composer require` both of them simultaneously. I'm developing a commerce integration using v3, but we also rely on a third-party plugin that's locked into v2.

**Comment by mrweiner** [comment]
If Recurly could provide a second packagist package tied to the v2 tree then this repo and https://packagist.org/packages/recurly-v2/recurly-clientv2 would be unnecessary.
